packages/masks/src/constants.ts

```diff
@@ -95,8 +95,7 @@ const ICON_CLIP_PATHS = {
 
 const CLIP_PATHS = {
   [MaskTypes.RECTANGLE]: 'M 0,0 1,0 1,1 0,1 0,0 Z',
-  [MaskTypes.CIRCLE]:
-    'M 0.972222 , 0.500000 c 0.000000 , 0.261111 -0.211111 , 0.472222 -0.472222 , 0.472222 S 0.027778 , 0.761111 , 0.027778 , 0.500000 S 0.238889 , 0.027778 , 0.500000 , 0.027778 S 0.972222 , 0.238889 , 0.972222 , 0.500000 z',
+  [MaskTypes.CIRCLE]: 'M 0,0.5 a 0.5,0.5 0 1,0 1,0 a 0.5,0.5 0 1,0 -1,0',
```

Comment:
(repeating my comment from the PR description to allow for an inline conversation)

This would change every story with a circle in it, making it slightly larger than it was when originally designed. Not sure this is undesirable.

We can merge this as-is or do an alternative:

1. Merge as-is impacting every story and template with a circle mask
2. Make a migration slightly reducing the size of all circle masks already used in a story (would probably be imprecise, but could come close)
3. Keep the old circle as it was, retire the mask, and a new circle (CIRCLE2) with the proper dimensions, only show the new circle in the mask library. All old stories would remain unchanged and all new stories would only use the new "proper" circle.

WDYT, @miina, @swissspidy?

Reply:
Option 3 seems the safest/cleanest to me.

Reply:
Yeah let's go with option 3. We already have `showInLibrary` so adding this as a new `CIRCLE_2` should be pretty straightforward.

Can add a comment to the old one pointing to the ticket.

<table>
<tr>
	<td>Old
	<td>New
<tr>
	<td><img width="344" alt="Screenshot 2022-12-22 at 09 36 48" src="https://user-images.githubusercontent.com/841956/209098494-a60d138a-3e70-44f5-88b5-012da96ea04e.png">
	<td><img width="341" alt="Screenshot 2022-12-22 at 09 36 56" src="https://user-images.githubusercontent.com/841956/209098482-09cb95f1-0d28-4e24-ab5e-812a7aac80b6.png">
</table>

Reply:
Done in e7b6ff2f00.